requirements.txt

```diff
@@ -13,5 +13,5 @@ Werkzeug==0.14.1
 yamlordereddictloader==0.4.0
 pyyaml==4.2b4
 raven[flask]==6.10.0
-prometheus_client==0.3.1
+prometheus_client==0.4.2
```

Comment:
let's keep `prometheus_client==0.3.1` for now and check if we need to have it in the requirements